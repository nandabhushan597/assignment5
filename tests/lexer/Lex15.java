12.34
12.
  1_2.3_4
  12e3
  12e-3
  12f
  .123e3
  .123e3f
  .123ef

  
