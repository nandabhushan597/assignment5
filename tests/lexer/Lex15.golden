Right [LiteralT (DoubleL 12.34)
LiteralT (DoubleL 12.0)
LiteralT (DoubleL 12.34)
LiteralT (DoubleL 12000.0)
LiteralT (DoubleL 1.2e-2)
LiteralT (FloatL 12.0)
LiteralT (DoubleL 123.0)
LiteralT (FloatL 123.0)
LiteralT (DoubleL 0.123)
IdentifierT "ef"]